docs/Marketplace.md

```diff
@@ -0,0 +1,360 @@
+
+**Storage:**
+```text
+address: 0x45BC5ca0fbdD9F920Edd12B90908448C30F32a37
+private key: 0x06c7ac11d4ee1d0ccb53811b71802fa92d40a5a174afad9f2cb44f93498322c3
+```
+**Client:**
+```text
+address: 0x9F0C62Fe60b22301751d6cDe1175526b9280b965
+private key: 0x5538ec03c956cb9d0bee02a25b600b0225f1347da4071d0fd70c521fdc63c2fc
+```
+
+### 2.3. Provision Accounts with Tokens
+
+We now need to transfer some ETH to each of the accounts, as well as provide them with some Codex tokens for the storage node to use as collateral and for the client node to buy actual storage.
+
+Although the process is not particularly complicated, I suggest you use [the script we prepared](https://github.com/gmega/local-codex-bare/blob/main/scripts/mint-tokens.js) for that. This script, essentially:
+
+1. reads the Marketplace contract address and its ABI from the deployment data;
+2. transfers $1$ ETH from the signer account to a target account if the target account has no ETH balance;
+3. mints $n$ Codex tokens and adds it into the target account's balance.
+
+To use the script, just copy it locally into a file named `mint-tokens.js`, and run:
+
+```bash
+# Installs Web3-js
+npm install web3
+# Provides tokens to the storage account.
+node ./mint-tokens.js <signer account address> 0x45BC5ca0fbdD9F920Edd12B90908448C30F32a37 10000000000
+# Provides tokens to the client account.
+node ./mint-tokens.js <signer account address> 0x9F0C62Fe60b22301751d6cDe1175526b9280b965 10000000000
+```
+
+Don't forget to replace `<signer account address>` with the address of the signer account you created in Step 1.1.
+
+## 3. Run Codex
+
+With accounts and geth in place, we can now start the Codex nodes.
+
+### 3.1. Storage Node
+
+The storage node will be the one storing data and submitting the proofs of storage to the chain. To do that, it needs access to:
+
+1. the address of the Marketplace contract that has been deployed to the local geth node in [Step 2.1](#21-deploy-the-codex-marketplace-contracts);
+2. the sample ceremony files which are shipped in the Codex contracts repo.
+
+Recall you have clone the `codex-contracts-eth` repository in Step 2.1. All of the required files are in there.
+
+**Address of the Marketplace Contract.** The contract address can be found inside of the file `codex-contracts-eth/deployments/codexdisttestnetwork/Marketplace.json`:
+
+```bash
+> grep '"address":' Marketplace.json
+  "address": "0x8891732D890f5A7B7181fBc70F7482DE28a7B60f",
+```
+
+**Prover ceremony files.** The ceremony files are under the `codex-contracts-eth/verifier/networks/codexdisttestnetwork` subdirectory. There are three of them: `proof_main.r1cs`, `proof_main.zkey`, and `prooof_main.wasm`. We will need all of them to start the Codex storage node.
+
```

Comment:
```suggestion

```sh
export CONTRACT_DEPLOY_FULL="codex-contracts-eth/deployments/codexdisttestnetwork"
export CONTRACT_VERIFIER=codex-contracts-eth/verifier/networks/codexdisttestnetwork/
```

Reply:
I don't know how to quote the backticks inside the backticks. 😢